varinfo/cmr_search.py

```diff
@@ -15,44 +15,57 @@
                                 DirectoryCreationException)
 
 
+CmrEnvType = Literal[CMR_OPS, CMR_UAT, CMR_SIT]
```

Comment:
I vaguely remember a conversation from a prior PR about wanting a type that shows the CMR environments aren't just any strings, but are specific strings. I did a bit of digging around, and that's what `typing.Literal` seems to do. (Was pretty happy with this find!)